src/apps/relay/prom_server.h

```diff
@@ -59,8 +59,8 @@ void start_prometheus_server(void);
 
 void prom_set_finished_traffic(const char* realm, const char* user, unsigned long rsvp, unsigned long rsvb, unsigned long sentp, unsigned long sentb, bool peer);
 
-void prom_inc_allocation(void);
-void prom_dec_allocation(void);
+void prom_inc_allocation(SOCKET_TYPE type);
+void prom_dec_allocation(SOCKET_TYPE type);
 #else
 
 void start_prometheus_server(void);
```

Comment:
I think this method does not need to be guarded by the `TURN_NO_PROMETHEUS ` define.

Reply:
I agree - we should have minimal amount if ifdefs outside of prom_server.c file to improve readability of the code